integration_tests/e2e/repeatCourtCaseJourney.cy.ts

```diff
@@ -508,7 +512,7 @@ context('Repeat Court Case journey', () => {
     Page.verifyOnPageTitle(CourtCaseConfirmationPage, 'Appearance')
   })
 
-  it('remand to sentencing journey with only one outcome option', () => {
+  it.skip('remand to sentencing journey with only one outcome option', () => {
```

Comment:
Might be worth adding a comment here in the code explaining why it's been skipped